Cargo.toml

```diff
@@ -24,6 +24,7 @@ log = "0.3"
 env_logger = "0.3"
 toml = { version = "0.2", features = ["serde"] }
 open = "1.1"
+regex = "0.1.80"
```

Comment:
It was the version that was already in cargo.lock; I'd be happy to upgrade it too, but I was trying to not mess with dependencies 😄